modules/core/core-project/src/main/java/com/enonic/xp/core/impl/project/ProjectServiceImpl.java

```diff
@@ -582,12 +622,57 @@ private <T> T callWithDeleteContext( final Callable<T> runnable, final ProjectNa
         return projectPermissionsContextManager.initDeleteContext( projectName ).callWith( runnable );
     }
 
+    private PropertyTree createContentRootData( final CreateProjectParams params )
+    {
+        PropertyTree data = new PropertyTree();
+
+        final PropertySet projectData = data.addSet( "data" );
+
+        if ( !params.getSiteConfigs().getApplicationKeys().isEmpty() )
+        {
+            SITE_CONFIGS_DATA_SERIALIZER.toProperties( params.getSiteConfigs(), projectData );
+        }
+
+        return data;
+    }
+
+    private PropertyTree getContentRootData( final ProjectName projectName )
+    {
+        if ( !this.repositoryService.isInitialized( projectName.getRepoId() ) )
```

Comment:
Strange to see isInitialazed here. It is supposed to be called from initializer only